src/settings/general-settings.tsx

```diff
@@ -34,6 +34,23 @@ const GeneralSettings: FC = () => {
 			}
 		}));
 	}, []);
+	const removeMod = useCallback((type: 'props' | 'prefs', key) => {
```

Comment:
key is an implicit any here. Also, I would type `type` as keyof Mods, but we can discuss about this